File: POO/poo11048216/Aulas/aula18.2/LDE/Item.cpp
#include "Item.h"

Item::Item()
{   }

Item::Item(int ch)
{
    this->setChave(ch);
    this->setProximo(NULL);
}

void Item::setChave(int ch)
{
    this->chave = ch;
}

void Item::setProximo(Item *it)
{
    this->proximo = it;
}

int Item::getChave()
{
    return this->chave;
}

Item * Item::getProximo()
{
    return this->proximo;
}

File: POO/poo11048216/Aulas/aula18.2/JLDE/Item.java
class Item
{
    private int chave;
    private Item proximo;
    
    public Item()
    {   }
    
    public Item(int ch)
    {
        this.setChave(ch);
        this.setProximo(null);
    }
    
    public void setChave(int ch)
    {
        this.chave = ch;
    }
    
    public void setProximo(Item it)
    {
        this.proximo = it;
    }
    
    public int getChave()
    {
        return this.chave;
    }
    
    public Item getProximo()
    {
        return this.proximo;
    }
}